entrypoint.sh

```diff
@@ -159,6 +169,34 @@ function get_files_matching_filetype() {
 	echo "${matching_files[@]}"
 }
 
+function load_filetype_array() {
```

Comment:
Since there was more in seiso_lint than just linting, I needed a way to recreate the linter objects without duplicating a lot of code. This is a start. I plan on cleaning up and making functions of a few more aspects in future sprints.

Reply:
Sounds good